docs/SRS-Volere/SRS.tex

```diff
@@ -77,22 +79,44 @@ \subsection{Maintenance Users and Service Technicians}
 
 \section{Mandated Constraints}
 \subsection{Solution Constraints}
-\lips
+\textbf{Mobile Device}\\
+\underline{Description}: User will interact with the system through an app on their mobile device (smartphone).\\
+\underline{Rational}: Mobile devices are widly used and ubiquitous. While at a grocery store, users may need access to the system.\\
+\underline{Fit Criterion}: User interface will run entirely on a mobile device.\\
+\\
+\textbf{Internet access}\\ %%todo: is this too solution specific?
+\underline{Description}: User must have internet access.\\
+\underline{Rational}: System servers and databases are used to store collective user data and process that data. User's devices are required to communicate to with the servers and databases via internet access.\\
```

Comment:
- User's devices are required to communicate **to with** the servers and databases via internet access. I think you need to delete the to in this sentence.